package.json

```diff
@@ -8,6 +8,8 @@
   "main": "./source/index.js",
   "bin": "./source/index.js",
   "scripts": {
+    "create-example": "react-native init example && cd example && yarn add -D react-native-clean-project",
```

Comment:
Just curious on why is this necessary?

Reply:
this was necessary because I saw a test entry in there but couldn't see how you were doing tests in here, so I went with ad-hoc testing using a quick example app I could sync changes into and run. Not automated but it is a very quick test/debug cycle, I must have run it 50 times while slowly making changes.

You mention tapejs, would you be willing to put an example of how you want to test things into master, then I can copy that pattern and exercise the changes?

Or if it is already there and I'm mixing something obvious just clue me in and I'll get to it

Reply:
You're right, there isn't a setup to run tests yet (sorry for that). Let me setup it and put it in master. I'll go with jest instead. 

I'll write here as soon as it's done.

Reply:
@mikehardy setup for running tests and a small test is now on master.

Reply:
Fantastic, I'll get after it today